test/backend/DataVaultTestSuite.scala

```diff
@@ -134,6 +134,8 @@ class DataVaultTestSuite extends PlaySpec {
       class MockDataVault extends DataVault {
         override def readBytesAndEncoding(path: VaultPath, range: RangeSpecifier)(
             implicit ec: ExecutionContext): Fox[(Array[Byte], Encoding.Value)] = ???
+
+        override def listDirectory(path: VaultPath,maxItems: Int)(implicit ec: ExecutionContext): Fox[List[VaultPath]] = ???
```

Comment:
```suggestion
        override def listDirectory(path: VaultPath, maxItems: Int)(implicit ec: ExecutionContext): Fox[List[VaultPath]] = ???
```
did the formater not lament this? :thinking:

Reply:
I guess the tests aren’t covered by the formatter config. That’s not a big deal imo :shrug: